libtransmission/ptrarray.h

```diff
@@ -47,7 +47,7 @@ void tr_ptrArrayForeach(tr_ptrArray* array, PtrArrayForeachFunc func);
 
 /** @brief Return the nth item in a tr_ptrArray
     @return the nth item in a tr_ptrArray */
-static inline void* tr_ptrArrayNth(tr_ptrArray* array, int i)
+static inline void* tr_ptrArrayNth(const tr_ptrArray* array, int i)
```

Comment:
Leaving this as non-const was intentional because the return value isn't const.

If you're trying to do this to loop through a const array of labels, check out how `tr_ptrArrayBase()` is used e.g. in session.c:

```c
    if (!tr_ptrArrayEmpty(&loadme))
    {    
        int const n = tr_ptrArraySize(&loadme);
        char const* const* paths = (char const* const*)tr_ptrArrayBase(&loadme);

        for (int i = 0; i < n; ++i) 
        {
            tr_list_append(&blocklists, tr_blocklistFileNew(paths[i], isEnabled));
        }
    }    
```

Reply:
Thanks for the suggestion, yes that was my intent.